src/main/scala/com/cloudera/sparkts/ARIMA.scala

```diff
@@ -607,7 +602,6 @@ class ARIMAModel(
       // copy into results
       results(-(d + nFuture) to -1) := forwardIntegrated
     }
-    results
```

Comment:
I think this was a bug as it caused this to always be the return value, when the else branch above intended to return something else? or do I misunderstand, which case the if branch is redundant?